plugins/generic/externalFeed/classes/ExternalFeedDAO.inc.php

```diff
@@ -63,11 +77,8 @@ function getExternalFeedJournalId($feedId) {
 	 * @param $row array
 	 * @return ExternalFeed
 	 */
-	function &_returnExternalFeedFromRow($row) {
-		$externalFeedPlugin =& PluginRegistry::getPlugin('generic', $this->parentPluginName);
-		$externalFeedPlugin->import('ExternalFeed');
-
-		$externalFeed = new ExternalFeed();
+	public function _returnExternalFeedFromRow($row) {
```

Comment:
Our newer conventions use a standard `_fromRow` rather than a different-for-every-class `_returnXYZFromRow`.